src/index.html

```diff
@@ -188,6 +188,12 @@
         }
     </style>
 
+    {{#if isProduction}}
```

Comment:
Add function "isProduction" to WavesApp for use in AppConfig and AppRun (initialize log level for i18next...)